docs/Makefile

```diff
@@ -7,13 +45,6 @@ SPHINXBUILD   = sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 
-# Put it first so that "make" without argument is like "make help".
+.PHONY: help Makefile
```

Comment:
I realize that you didn't change this, but I'll just observe that it seems weird to declare `Makefile` as a phony target.  For one thing, I don't see a definition for a `Makefile` target, but, more importantly, if we _were_ to produce a makefile, then we probably don't want to reproduce it unless it is stale, which is part-and-parcel to Make's handling of (non-phony) targets.

Reply:
I presume that this came with the Sphinx stuff, but I have no recollection of doing that. Maybe some git archaeology is warranted to find out when/who/how (if it comes back to me, I'll take the Reagan defence).